atd-etl/prefect/afd_ems_incidents/afd_incident_upload.py

```diff
@@ -0,0 +1,279 @@
+
+        sql = ""
+        values = []
+        if existing["existing"] > 0:
+            # print("Updating existing record")
+            sql = """
+                update afd__incidents set
+                    unparsed_ems_incident_number = %s,
+                    ems_incident_numbers = %s,
+                    call_datetime = %s,
+                    calendar_year = %s,
+                    jurisdiction = %s,
+                    address= %s,
+                    problem = %s,
+                    flagged_incs = %s,
+                    geometry = ST_SetSRID(ST_MakePoint(%s, %s), 4326)
+                where incident_number = %s
+            """
+
+            values = [
+                row["EMS_IncidentNumber"],
+                "{" + ", ".join(ems_numbers) + "}",
+                row["Inc_Date"].strftime("%Y-%m-%d")
+                + " "
+                + row["Inc_Time"].strftime("%H:%M:%S"),
+                row["CalendarYear"],
+                row["Jurisdiction"],
+                row["CAD_Address"],
+                row["CAD_Problem"],
+                row["Flagged_Incs"],
+                longitude,
+                latitude,
+                row["Incident_Number"],
+            ]
+        else:
+            # print("Inserting new record")
+            sql = """
+                insert into afd__incidents (
+                    incident_number, 
+                    unparsed_ems_incident_number, 
+                    ems_incident_numbers,
+                    call_datetime, 
+                    calendar_year, 
+                    jurisdiction, 
+                    address, 
+                    problem, 
+                    flagged_incs, 
+                    geometry
+                ) values (%s, %s, %s, %s, %s, %s, %s, %s, %s, ST_SetSRID(ST_Point(%s, %s), 4326))
+                ON CONFLICT DO NOTHING;
+            """
+
+            values = [
+                row["Incident_Number"],
+                row["EMS_IncidentNumber"],
+                "{" + ", ".join(ems_numbers) + "}",
+                row["Inc_Date"].strftime("%Y-%m-%d")
+                + " "
+                + row["Inc_Time"].strftime("%H:%M:%S"),
+                row["CalendarYear"],
+                row["Jurisdiction"],
+                row["CAD_Address"],
+                row["CAD_Problem"],
+                row["Flagged_Incs"],
+                longitude,
+                latitude,
+            ]
+
+        try:
+            cursor = pg.cursor()
+            cursor.execute(sql, values)
+            pg.commit()
+        except Exception as error:
+            print(f"Error executing:\n\n{sql}\n")
+            print(f"Values: {values}")
+            print(f"Error: {error}")
+            pg.rollback()
+
+    return True
+
+
+@task
+def clean_up(path):
+    # Clean up the temp location
+    shutil.rmtree(path)
+
+
+with Flow("AFD Import ETL") as flow:
+    record_age_maximum = Parameter("record_age_maximum", default=False)
```

Comment:
This parameter is of type `int`, the default should be `90`.

Reply:
setting it as False gets a full import? so we default to a full import?

Reply:
Thank you for catching that! I updated the PR to have a default of `0` for both flows' value for the `record_age_maximum` int parameter. This will make the default behavior to do a full import.